README.md

```diff
@@ -1,3 +1,5 @@
 # Git Good
 
-The repo is designed to help people learn effective git branching strategies when working on a team.
\ No newline at end of file
+The repo is designed to help people learn effective git branching strategies when working on a team.
+
+This is comment made by Mitchell Keo to practice git branching :O !
```

Comment:
Unusable. You'll need to add "a" after "This is".

Reply:
F